tests/functional/lib/helpers.js

```diff
@@ -49,90 +44,68 @@ define(
       );
     }
 
-    // TODO: when https://github.com/mozilla/geckodriver/issues/308 is fixed,
-    // remove this ugliness.
-
-    function login(context) {
```

Comment:
Can you file a follow up issue to create a way to test logging in, please?

Reply:
I think we already have and issues like that, wait...
Yeah! #1385

Reply:
Ah, perfect. Thanks!